tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -188,9 +185,43 @@ def test_instantiation_with_temp_table_schema():
             create_temp_table=True,
             temp_table_schema_name="test_schema",
         )
-        query_to_create_temp_table: str = batch_data._create_temporary_table(
-            temp_table_name="hello",
+        (
+            query_to_create_temp_table,
+            temp_table_name,
+        ) = batch_data._create_temporary_table(
+            dialect=GXSqlDialect.SQLITE,
             query="test_query",
             temp_table_schema_name="test_schema",
         )
         assert "test_schema" in query_to_create_temp_table
+
+
+def test_instantiation_with_selectable_only_and_no_temp_table(sqlite_view_engine, sa):
+    """
+    What does this test and why?
+
+    In cases where we create a validator but explicitly set `create_temp_table`=False, we directly use the
+    selectable created by SqlAlchemyExecutionEngine's _build_selectable_from_batch_spec() method.
+    """
+
+    selectable = sa.select("*").select_from(sa.text("main.test_table"))
+    print(get_sqlite_temp_table_names_from_engine(sqlite_view_engine))
```

Comment:
Could we remove this print statement from the test just to clean up the test output?

Reply:
Yes definitely. Thanks for the catch.